config/v1/types_oauth.go

```diff
@@ -48,7 +48,17 @@ type TokenConfig struct {
 	// accessTokenMaxAgeSeconds defines the maximum age of access tokens
 	AccessTokenMaxAgeSeconds int32 `json:"accessTokenMaxAgeSeconds"`
 
-	// accessTokenInactivityTimeoutSeconds - DEPRECATED: setting this field has no effect.
+	// accessTokenInactivityTimeoutSeconds defines the default token
+	// inactivity timeout for tokens granted by any client.
+	// The value represents the maximum amount of time that can occur between
+	// consecutive uses of the token. Tokens become invalid if they are not
+	// used within this temporal window. The user will need to acquire a new
+	// token to regain access once a token times out.
+	// Valid values are integer values:
+	//   x < 0  Tokens time out is enabled but tokens never timeout unless configured per client (e.g. `-1`)
+	//   x = 0  Tokens time out is disabled (default)
+	//   x > 0  Tokens time out if there is no activity for x seconds
+	// The current minimum allowed value for X is 300 (5 minutes)
 	// +optional
 	AccessTokenInactivityTimeoutSeconds int32 `json:"accessTokenInactivityTimeoutSeconds,omitempty"`
```

Comment:
Seems like users would prefer to enter a string duration `2d12h5m` as opposed to int seconds.  why not simply retire the field entirely and create that?

Reply:
@deads2k Added a new field `AccessTokenInactivityTimeout` of type `metav1.Duration`. Please take a look.